parcel.go

```diff
@@ -2,59 +2,92 @@ package main
 
 import (
 	"database/sql"
+	"errors"
 )
 
 type ParcelStore struct {
 	db *sql.DB
 }
 
-func NewParcelStore(db *sql.DB) ParcelStore {
-	return ParcelStore{db: db}
+func NewParcelStore(db *sql.DB) *ParcelStore {
+	return &ParcelStore{db: db}
 }
 
-func (s ParcelStore) Add(p Parcel) (int, error) {
-	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
-	// верните идентификатор последней добавленной записи
-	return 0, nil
+func (s *ParcelStore) Add(p Parcel) (int, error) {
```

Comment:
Здесь тоже не нужно принимать по ссылке везде
```suggestion
func (s ParcelStore) Add(p Parcel) (int, error) {
```